### PR TITLE
Pfeiffer master

### PR DESCRIPTION
**Pull request by ladanazita**
@hankim813 @pfeiffer @anoonan @Peripheral1994 

Resolved conflicts for pull/26 on a new branch. Updates History.md and package.json to release 2.4.1

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2.4.1 / 2017-03-08
```
Comment (hankim813) [inline HISTORY.md]: should be 2.4.0

**Review by hankim813 — APPROVED** [review]
nit pick on version and can you guys e2e test this real quick before deploying?

**Comment by ladanazita** [comment]
e2e tested 👍 : 
image

This closes pull #19

**Comment by ladanazita** [comment]
@pfeiffer this is now live 😄 thanks for the PR!

**Comment by pfeiffer** [comment]
Awesome! Thanks :-)

**Pull request merged**